pkg/operator/ceph/object/rgw.go

```diff
@@ -63,6 +63,8 @@ type rgwConfig struct {
 
 const (
 	oldRgwKeyName = "client.radosgw.gateway"
+	// PrefixObjectStoreSvc is the DNS prefix of the object store service
+	PrefixObjectStoreSvc = "rook-ceph-rgw"
```

Comment:
Why not use `appName`?

Reply:
I was just moving some code around but you're right let's just use `appName` since it's identical.